sdk/program/src/stable_layout.rs

```diff
@@ -0,0 +1,7 @@
+//! bprumo TODO: doc, or hide doc?
+
```

Comment:
/// DO NOT USE THIS MODULE. FOR INTERNAL USE ONLY. IF YOU USE THIS MODULE ALL
/// YOUR APES GONE.